pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -285,21 +307,21 @@ func newBackendDeployment() *appsv1.Deployment {
 		Replicas: &replicas,
 		Selector: &metav1.LabelSelector{
 			MatchLabels: map[string]string{
-				"app": serviceName,
+				"app": name,
```

Comment:
Is there a reason not to use `app.kubernetes.io/name` here?

(If not, you'll need to update the code below too).

Reply:
8c691fd thank you, fixed.